### ISSUE TITLE
Images must have alternate text

### ISSUE THREAD

**Issue by christinach** [pulibrary/static-tables#97]
### Steps to recreate
Go to https://static-tables-staging.princeton.edu/
Open the dev console.
Scan the page using axe-core tools browser extension

### URL
https://static-tables-staging.princeton.edu/

### Issues
Ensures <img> elements have alternate text or a role of none or presentation

### Severity
Critical

### WCAG Success Criterion
[WCAG 1.1.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Screen readers have no way of translating an image into words that gets read to the user, even if the image only consists of text. As a result, it's necessary for images to have short, descriptive alt text so screen reader users clearly understand the image's contents and purpose.

If you can't see, all types of visual information, such as images, are completely useless unless a digital text alternative is provided so that screen readers can convert that text into either sound or braille. The same is true in varying degrees for people with low vision or color-blindness.

When you do not provide an acceptable alternative that works for their available sensory modalities, such as making an image accessible by providing a digital text description, screen readers cannot convert it into speech or braille to make it available by sound or touch. (Reference [dequeuniversity.com/image-alt](https://dequeuniversity.com/rules/axe/4.4/image-alt))

The intent of this Success Criterion is to make information conveyed by non-text content accessible through the use of a text alternative. Text alternatives are a primary way for making information accessible because they can be rendered through any sensory modality (for example, visual, auditory or tactile) to match the needs of the user. Providing text alternatives allows the information to be rendered in a variety of ways by a variety of user agents. For example, a person who cannot see a picture can have the text alternative read aloud using synthesized speech. A person who cannot hear an audio file can have the text alternative displayed so that he or she can read it. In the future, text alternatives will also allow information to be more easily translated into sign language or into a simpler form of the same language. (Reference: [WCAG SC 1.1.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html))

### Recommended fix
You need to fix at least (1) of the following:
Element does not have an alt attribute
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element's default semantics were not overridden with role="none" or role="presentation"

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
[Accessibility audit_05152024_Static-Tables](https://docs.google.com/spreadsheets/d/1_-Khj5TezyEu2gVb2jJczWCUt04bt5_P8-zL0f269kQ/edit#gid=0)

**Comment by sandbergja** [comment]
This has been fixed!

**Comment by kevinreiss** [comment]
I will close the issue, please re-open if I'm misunderstanding.
